website/members/api/v2/views.py

```diff
@@ -46,3 +47,8 @@ class MemberDetailView(RetrieveAPIView):
         DjangoModelPermissionsOrAnonReadOnly,
     ]
     required_scopes = ["activemembers:read"]
+
+
+class MemberCurrentView(MemberDetailView):
```

Comment:
Shouldn't this require the member profile read scope?

Reply:
I was thinking of using the same scope as regular users. But actually a good idea to make it more restrictive since an app might not need access to the full members database. Good catch.

Also, I noted that the scope of this piece of code is wrong anyways.